### PR TITLE
Add ts support in amagaki projects

### PR DESCRIPTION
**Pull request by uxder**
- support correct ts loading
- minimization on `npm run build`
- support for live reloading

### CODE REVIEW

File: amagaki.js [hunk 1]
```diff
@@ -18,6 +18,10 @@ module.exports = function (pod) {
         path: '/static/css/',
         staticDir: '/dist/css/',
       },
+      {
+        path: '/static/',
```
Comment (jeremydw) [inline amagaki.js:22]: I think we should combine this with the previous line? So I believe you can delete the lines 17-20 in favor of this updated rule.

**Comment by jeremydw** [comment]
Is this ready to go? If so I think it will fix https://github.com/blinkkcode/amagaki-starter/issues/2

**Comment by uxder** [comment]
Hey Jeremy, - It's working but the live reload is delayed by about one or two seconds (minor issue).

**Comment by jeremydw** [comment]
Pulled this in from your work on another project; closing this PR

**Pull request closed without merging**